panels/zwave/ha-panel-zwave.html

```diff
@@ -91,7 +112,319 @@
           </div>
         </paper-card>
       </div>
-    </app-header-layout>
+
+      <div class='content'>
+        <paper-card heading='Z-Wave Node Management'>
+          <div class='card-content'>
+            Z-Wave Node controls.
+          </div>
+          <div class='device-picker'>
+            <paper-dropdown-menu label="Nodes" class="flex">
+              <paper-listbox
+                class="dropdown-content"
+                selected='{{selectedNode}}'>
+              <template is='dom-repeat' items='[[nodes]]' as='state'>
+                <paper-item>[[computeSelectCaption(state)]]</paper-item>
+              </template>
+              </paper-listbox>
+            </paper-dropdown-menu>
+          </div>
+
+          <template is='dom-if' if='[[!computeIsNodeSelected(selectedNode)]]'>
+          <div class='card-actions'>
+            <ha-call-service-button
+                hass='[[hass]]'
+                domain='zwave'
+                service='refresh_node'
+                service-data=[[computeNodeServiceData(selectedNode)]]
```

Comment:
If you're going to compute the same service data many times, put it in a computed property so it is only computed once.